### ISSUE TITLE
Bug: season games are not saved properly

### ISSUE THREAD

**Issue by Helias** [azerothcore/mod-arena-3v3-solo-queue#25]
https://github.com/chromiecraft/chromiecraft/issues/7496

**Comment by Apathyxia** [comment]
I'd like to comment here that it seems like replaying games also adds to the number of games played on the website. I think these are counting as losses. 

Also, the way required games for points are shown is not correct. A person does not always need 10 games. The team needs at least 10 games, and each person needs to be part of >=30% games that week to be eligible to receive points. 

I checked on my team's games when I had done 6 games and my team had 18 games at that point that the website showed me to require 4 more games (I did those just in case something changed in the code how points are distributed). 6 games out of 18 total should be enough to be eligible for points. 

And, if I remember correctly, after 1500 team rating, a person needs to be within 150 rating of the team's rating on top of the above requirement.
